.github/CHANGELOG.md

```diff
@@ -92,6 +92,9 @@
 
 ### Bug fixes
 
+* Update CUDA version to 12.4 for GPU tests on CI.
+  [(#1023)](https://github.com/PennyLaneAI/pennylane-lightning/pull/1018)
```

Comment:
```suggestion
  [(#1023)](https://github.com/PennyLaneAI/pennylane-lightning/pull/1023)
```

Reply:
Thanks Ali ... this bug has really destroyed me🤦